### DISCUSSION TITLE
APK Editor Studio v1.6.0

### DISCUSSION THREAD

**Discussion by kefir500** [kefir500/apk-editor-studio#53] [Releases]
- Added Smali syntax highlighting.
- Added Find and Replace features to code editor.
- Added code folding.
- Added code indentation control (Tab, Shift+Tab).
- Added custom dark theme for all platforms.
- Added stand-alone version of Android Explorer.
- Added ability to filter resource tree.
- Added ability to build debuggable APKs.
- Added Android 12 support.
- Updated Apktool to 2.6.0.
- Migrated to a new syntax highlighter.
- Set AAPT2 as a default option.
- Remember resource tree columns state.
- Remove empty resource tree nodes.
- Improved Android Explorer window.
- Improved dynamic GUI translation.
- Fixed erroneous resource item deletion on open file.
- Added explicit warning for default keystore.
- Windows: Fixed WebP image format support.
- Windows: Changed default editor font to Consolas.
- Linux: Dropped DEB package.
- Linux: Updated Qt to 5.15.2.

**Comment by sonwnja** [comment]
终于看到你们更新了，也支持smali编辑了，希望后续能加入smali2java的功能，你们做的很好，真的，如果赞助方式可以选择支付宝的话我会赞助你们，paypal让我感觉非常不方便

**Reply by kefir500** [reply]
Hi, thank you for the feedback!
Yes, I have plans to add a Java decompiler.
Regarding the Alipay. Unfortunately, this payment system isn't supported in my country.

**Comment by Ibuprophen** [comment]
Chinese to English: I finally saw that you updated and support smali editing. I hope to add the function of smali2java in the future. You have done a good job. Really, I will sponsor you if you can choose Alipay. Paypal makes me feel very inconvenient.

**Reply by NannyCuore** [reply]
hoy es la primera vez que uso su proyecto se ve muy bien felicidades yo la verdad soy nuevo en esto pero me gusta aprender saludos desde mexico
